.github/ISSUE_TEMPLATE/cm---issue-template.md

```diff
@@ -0,0 +1,30 @@
+---
+name: CM - Issue template
+about: Bug report
+title: [BUG] Please provide a title 
+labels: 'bug'
+assignees: ''
+
+---
+
+I'm submitting a bug report.
```

Comment:
Line 10 will be line 1, and if the top part is not visible, nothing will mention it's a bug report. Are the questions enough to make it look like a bug report? If yes, happy to remove it.

Reply:
Once you create an issue with the template - you will no longer see what type of template was used to create it. This is how it will look --> [here](https://github.com/CodeforAustralia/council-emissions-calculator/issues/213#issue-1402555884)